### ISSUE TITLE
Request: Validation of pickup point required as initial condition in checkout

### ISSUE THREAD

**Issue by mainiomooses** [Pakettikauppa/woo-pakettikauppa#160]
At checkout the pickup point is required and/or validated after the select fields become available. This means that if the customer is somehow able to have the checkout form filled without the AJAX call for updating the shipping options, he/she can proceed in the ordering process without ever having to select a pickup point - even though the selected shipping method would have required it.

This results in an order without a pickup point selected. Also, the default of "nearest" pickup point to the shipping address doesn't get applied as the selection data is never considered in this case. This prompts the seller having to contact the customer for this information that should have been collected at checkout.

My request is that the validation to be made whether or not the pickup point selections have been made available - that is, if a delivery method is one of Pakettikauppa's, the user cannot check out without having made a selection OR if no selection is made, the nearest pickup point is automatically assigned.

**TL;DR:** Make selecting a pickup point for all applicable delivery methods required and validated initially, not only after a ZIP code has been entered to avoid issues with AJAX.
